### PR TITLE
Fix miscellaneous types and casts

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
36 fewer warnings in Windows builds.

### CODE REVIEW

File: assignment-client/src/audio/AudioMixer.cpp [hunk 1]
```diff
@@ -60,7 +60,7 @@
 #include "AudioMixer.h"
 
 const float LOUDNESS_TO_DISTANCE_RATIO = 0.00001f;
-const float DEFAULT_ATTENUATION_PER_DOUBLING_IN_DISTANCE = 0.18;
+const float DEFAULT_ATTENUATION_PER_DOUBLING_IN_DISTANCE = 0.18f;
```
Comment (Atlante45) [inline assignment-client/src/audio/AudioMixer.cpp]: I'm feeling so ashamed right now!

File: libraries/audio/src/AudioConstants.h [hunk 2]
```diff
@@ -26,8 +26,8 @@ namespace AudioConstants {
     const int NETWORK_FRAME_BYTES_PER_CHANNEL = 512;
     const int NETWORK_FRAME_SAMPLES_PER_CHANNEL = NETWORK_FRAME_BYTES_PER_CHANNEL / sizeof(AudioSample);
     const float NETWORK_FRAME_MSECS = (AudioConstants::NETWORK_FRAME_SAMPLES_PER_CHANNEL
-        / (float) AudioConstants::SAMPLE_RATE) * 1000.0;
-    const unsigned int NETWORK_FRAME_USECS = floorf(NETWORK_FRAME_MSECS * 1000.0);
+        / (float)AudioConstants::SAMPLE_RATE) * 1000.0f;
+    const unsigned int NETWORK_FRAME_USECS = (int)floorf(NETWORK_FRAME_MSECS * 1000.0f);
```
Comment (AndrewMeadows) [inline libraries/audio/src/AudioConstants.h]: Perhaps this cast should be (unsigned int)?

File: tests/octree/src/OctreeTests.cpp [hunk 3]
```diff
@@ -802,7 +802,7 @@ void OctreeTests::propertyFlagsTests(bool verbose) {
             qDebug() << "fill encoded byte array with extra garbage (as if it was bitstream with more content)";
         }
         QByteArray extraContent;
-        extraContent.fill(0xba, 10);
+        extraContent.fill(0xbau, 10);
```
Comment (AndrewMeadows) [inline tests/octree/src/OctreeTests.cpp]: My preferred format would be 0xbaU here and 0xffU further up, however the coding standard is silent on this issue.  This doesn't block the merge -- I'm just feeling chatty.

**Comment by ctrlaltdavid** [comment]
Good points. Thanks @AndrewMeadows . Update coming forthwith.

**Pull request merged**